themes/bootstrap3/js/hierarchy_tree.js

```diff
@@ -0,0 +1,258 @@
+/*global VuFind */
+VuFind.register('hierarchyTree', function HierarchyTree() {
+  /* Utility functions */
+  function htmlEncodeId(id) {
+    return id.replace(/\W/g, "-"); // Also change hierarchy/tree-level.phtml
```

Comment:
What does this "Also change" comment mean? Is it a TODO?

Reply:
That should have read "If you change this, also change...". But I eliminated the function by getting rid of the `name` attribute and using `data-record-id` instead to find entries of records. This also eliminates the possibility of duplicate identifiers that `htmlEncodeId` had.